DB Schema

**Describe the bug**
LPVS doesn't run.

**To Reproduce**
Run LPVS along with the mysql server with the [schema](https://github.com/Samsung/LPVS/blob/main/src/main/resources/database_dump.sql) applied. 

**Error logs**
```
$ docker logs -f lpvs-lpvs-1 

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.7.9)

2023-05-08 06:37:10.972  INFO 1 --- [           main] com.lpvs.LicensePreValidationSystem      : Starting LicensePreValidationSystem v1.0.2 using Java 11.0.16 on 65c39866b683 with PID 1 (/LPVS/target/lpvs-1.0.2.jar started by root in /LPVS/target)
2023-05-08 06:37:10.977  INFO 1 --- [           main] com.lpvs.LicensePreValidationSystem      : No active profile set, falling back to 1 default profile: "default"
2023-05-08 06:37:11.811  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2023-05-08 06:37:11.913  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 87 ms. Found 5 JPA repository interfaces.
2023-05-08 06:37:12.667  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 7896 (http)
2023-05-08 06:37:12.682  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-05-08 06:37:12.682  INFO 1 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.71]
2023-05-08 06:37:12.772  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-05-08 06:37:12.772  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 1717 ms
2023-05-08 06:37:13.009  INFO 1 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2023-05-08 06:37:13.079  INFO 1 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 5.6.15.Final
2023-05-08 06:37:13.292  INFO 1 --- [           main] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2023-05-08 06:37:13.440  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2023-05-08 06:37:13.897  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2023-05-08 06:37:13.918  INFO 1 --- [           main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL8Dialect
2023-05-08 06:37:14.950  INFO 1 --- [           main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2023-05-08 06:37:14.966  INFO 1 --- [           main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2023-05-08 06:37:15.404  WARN 1 --- [           main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'gitHubWebhooksController' defined in URL [jar:file:/LPVS/target/lpvs-1.0.2.jar!/BOOT-INF/classes!/com/lpvs/controller/GitHubWebhooksController.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'LPVSQueueService' defined in URL [jar:file:/LPVS/target/lpvs-1.0.2.jar!/BOOT-INF/classes!/com/lpvs/service/LPVSQueueService.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'LPVSGitHubService' defined in URL [jar:file:/LPVS/target/lpvs-1.0.2.jar!/BOOT-INF/classes!/com/lpvs/service/LPVSGitHubService.class]: Unsatisfied dependency expressed through constructor parameter 3; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'LPVSPullRequestRepository' defined in com.lpvs.repository.LPVSPullRequestRepository defined in @EnableJpaRepositories declared on JpaRepositoriesRegistrar.EnableJpaRepositoriesConfiguration: Invocation of init method failed; nested exception is org.springframework.data.repository.query.QueryCreationException: Could not create query for public abstract java.util.Date com.lpvs.repository.LPVSPullRequestRepository.getNow(); Reason: null; nested exception is java.lang.NullPointerException
2023-05-08 06:37:15.404  INFO 1 --- [           main] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2023-05-08 06:37:15.408  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2023-05-08 06:37:15.428  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2023-05-08 06:37:15.430  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2023-05-08 06:37:15.446  INFO 1 --- [           main] ConditionEvaluationReportLoggingListener : 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2023-05-08 06:37:15.501 ERROR 1 --- [           main] o.s.boot.SpringApplication               : Application run failed

org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'gitHubWebhooksController' defined in URL [jar:file:/LPVS/target/lpvs-1.0.2.jar!/BOOT-INF/classes!/com/lpvs/controller/GitHubWebhooksController.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'LPVSQueueService' defined in URL [jar:file:/LPVS/target/lpvs-1.0.2.jar!/BOOT-INF/classes!/com/lpvs/service/LPVSQueueService.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'LPVSGitHubService' defined in URL [jar:file:/LPVS/target/lpvs-1.0.2.jar!/BOOT-INF/classes!/com/lpvs/service/LPVSGitHubService.class]: Unsatisfied dependency expressed through constructor parameter 3; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'LPVSPullRequestRepository' defined in com.lpvs.repository.LPVSPullRequestRepository defined in @EnableJpaRepositories declared on JpaRepositoriesRegistrar.EnableJpaRepositoriesConfiguration: Invocation of init method failed; nested exception is org.springframework.data.repository.query.QueryCreationException: Could not create query for public abstract java.util.Date com.lpvs.repository.LPVSPullRequestRepository.getNow(); Reason: null; nested exception is java.lang.NullPointerException
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:800) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:229) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1372) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1222) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:582) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:955) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:918) ~[spring-context-5.3.25.jar!/:5.3.25]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:583) ~[spring-context-5.3.25.jar!/:5.3.25]
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:147) ~[spring-boot-2.7.9.jar!/:2.7.9]
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:731) ~[spring-boot-2.7.9.jar!/:2.7.9]
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:408) ~[spring-boot-2.7.9.jar!/:2.7.9]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:307) ~[spring-boot-2.7.9.jar!/:2.7.9]
	at com.lpvs.LicensePreValidationSystem.main(LicensePreValidationSystem.java:33) ~[classes!/:1.0.2]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49) ~[lpvs-1.0.2.jar:1.0.2]
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:108) ~[lpvs-1.0.2.jar:1.0.2]
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58) ~[lpvs-1.0.2.jar:1.0.2]
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:65) ~[lpvs-1.0.2.jar:1.0.2]
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'LPVSQueueService' defined in URL [jar:file:/LPVS/target/lpvs-1.0.2.jar!/BOOT-INF/classes!/com/lpvs/service/LPVSQueueService.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'LPVSGitHubService' defined in URL [jar:file:/LPVS/target/lpvs-1.0.2.jar!/BOOT-INF/classes!/com/lpvs/service/LPVSGitHubService.class]: Unsatisfied dependency expressed through constructor parameter 3; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'LPVSPullRequestRepository' defined in com.lpvs.repository.LPVSPullRequestRepository defined in @EnableJpaRepositories declared on JpaRepositoriesRegistrar.EnableJpaRepositoriesConfiguration: Invocation of init method failed; nested exception is org.springframework.data.repository.query.QueryCreationException: Could not create query for public abstract java.util.Date com.lpvs.repository.LPVSPullRequestRepository.getNow(); Reason: null; nested exception is java.lang.NullPointerException
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:800) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:229) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1372) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1222) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:582) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1391) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1311) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:887) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:791) ~[spring-beans-5.3.25.jar!/:5.3.25]
	... 25 common frames omitted
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'LPVSGitHubService' defined in URL [jar:file:/LPVS/target/lpvs-1.0.2.jar!/BOOT-INF/classes!/com/lpvs/service/LPVSGitHubService.class]: Unsatisfied dependency expressed through constructor parameter 3; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'LPVSPullRequestRepository' defined in com.lpvs.repository.LPVSPullRequestRepository defined in @EnableJpaRepositories declared on JpaRepositoriesRegistrar.EnableJpaRepositoriesConfiguration: Invocation of init method failed; nested exception is org.springframework.data.repository.query.QueryCreationException: Could not create query for public abstract java.util.Date com.lpvs.repository.LPVSPullRequestRepository.getNow(); Reason: null; nested exception is java.lang.NullPointerException
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:800) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:229) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1372) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1222) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:582) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1391) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1311) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:887) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:791) ~[spring-beans-5.3.25.jar!/:5.3.25]
	... 39 common frames omitted
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'LPVSPullRequestRepository' defined in com.lpvs.repository.LPVSPullRequestRepository defined in @EnableJpaRepositories declared on JpaRepositoriesRegistrar.EnableJpaRepositoriesConfiguration: Invocation of init method failed; nested exception is org.springframework.data.repository.query.QueryCreationException: Could not create query for public abstract java.util.Date com.lpvs.repository.LPVSPullRequestRepository.getNow(); Reason: null; nested exception is java.lang.NullPointerException
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1804) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1391) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1311) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:887) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:791) ~[spring-beans-5.3.25.jar!/:5.3.25]
	... 53 common frames omitted
Caused by: org.springframework.data.repository.query.QueryCreationException: Could not create query for public abstract java.util.Date com.lpvs.repository.LPVSPullRequestRepository.getNow(); Reason: null; nested exception is java.lang.NullPointerException
	at org.springframework.data.repository.query.QueryCreationException.create(QueryCreationException.java:101) ~[spring-data-commons-2.7.8.jar!/:2.7.8]
	at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.lookupQuery(QueryExecutorMethodInterceptor.java:107) ~[spring-data-commons-2.7.8.jar!/:2.7.8]
	at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.lambda$mapMethodsToQuery$1(QueryExecutorMethodInterceptor.java:95) ~[spring-data-commons-2.7.8.jar!/:2.7.8]
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195) ~[na:na]
	at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133) ~[na:na]
	at java.base/java.util.Collections$UnmodifiableCollection$1.forEachRemaining(Collections.java:1054) ~[na:na]
	at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801) ~[na:na]
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[na:na]
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[na:na]
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[na:na]
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[na:na]
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[na:na]
	at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.mapMethodsToQuery(QueryExecutorMethodInterceptor.java:97) ~[spring-data-commons-2.7.8.jar!/:2.7.8]
	at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.lambda$new$0(QueryExecutorMethodInterceptor.java:87) ~[spring-data-commons-2.7.8.jar!/:2.7.8]
	at java.base/java.util.Optional.map(Optional.java:265) ~[na:na]
	at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.<init>(QueryExecutorMethodInterceptor.java:87) ~[spring-data-commons-2.7.8.jar!/:2.7.8]
	at org.springframework.data.repository.core.support.RepositoryFactorySupport.getRepository(RepositoryFactorySupport.java:365) ~[spring-data-commons-2.7.8.jar!/:2.7.8]
	at org.springframework.data.repository.core.support.RepositoryFactoryBeanSupport.lambda$afterPropertiesSet$5(RepositoryFactoryBeanSupport.java:323) ~[spring-data-commons-2.7.8.jar!/:2.7.8]
	at org.springframework.data.util.Lazy.getNullable(Lazy.java:231) ~[spring-data-commons-2.7.8.jar!/:2.7.8]
	at org.springframework.data.util.Lazy.get(Lazy.java:115) ~[spring-data-commons-2.7.8.jar!/:2.7.8]
	at org.springframework.data.repository.core.support.RepositoryFactoryBeanSupport.afterPropertiesSet(RepositoryFactoryBeanSupport.java:329) ~[spring-data-commons-2.7.8.jar!/:2.7.8]
	at org.springframework.data.jpa.repository.support.JpaRepositoryFactoryBean.afterPropertiesSet(JpaRepositoryFactoryBean.java:144) ~[spring-data-jpa-2.7.8.jar!/:2.7.8]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1863) ~[spring-beans-5.3.25.jar!/:5.3.25]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1800) ~[spring-beans-5.3.25.jar!/:5.3.25]
	... 64 common frames omitted
Caused by: java.lang.NullPointerException: null
	at org.springframework.data.jpa.repository.query.QueryUtils.createCountQueryFor(QueryUtils.java:607) ~[spring-data-jpa-2.7.8.jar!/:2.7.8]
	at org.springframework.data.jpa.repository.query.DefaultQueryEnhancer.createCountQueryFor(DefaultQueryEnhancer.java:49) ~[spring-data-jpa-2.7.8.jar!/:2.7.8]
	at org.springframework.data.jpa.repository.query.StringQuery.deriveCountQuery(StringQuery.java:119) ~[spring-data-jpa-2.7.8.jar!/:2.7.8]
	at org.springframework.data.jpa.repository.query.AbstractStringBasedJpaQuery.<init>(AbstractStringBasedJpaQuery.java:72) ~[spring-data-jpa-2.7.8.jar!/:2.7.8]
	at org.springframework.data.jpa.repository.query.NativeJpaQuery.<init>(NativeJpaQuery.java:53) ~[spring-data-jpa-2.7.8.jar!/:2.7.8]
	at org.springframework.data.jpa.repository.query.JpaQueryFactory.fromMethodWithQueryString(JpaQueryFactory.java:51) ~[spring-data-jpa-2.7.8.jar!/:2.7.8]
	at org.springframework.data.jpa.repository.query.JpaQueryLookupStrategy$DeclaredQueryLookupStrategy.resolveQuery(JpaQueryLookupStrategy.java:169) ~[spring-data-jpa-2.7.8.jar!/:2.7.8]
	at org.springframework.data.jpa.repository.query.JpaQueryLookupStrategy$CreateIfNotFoundQueryLookupStrategy.resolveQuery(JpaQueryLookupStrategy.java:253) ~[spring-data-jpa-2.7.8.jar!/:2.7.8]
	at org.springframework.data.jpa.repository.query.JpaQueryLookupStrategy$AbstractQueryLookupStrategy.resolveQuery(JpaQueryLookupStrategy.java:93) ~[spring-data-jpa-2.7.8.jar!/:2.7.8]
	at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.lookupQuery(QueryExecutorMethodInterceptor.java:103) ~[spring-data-commons-2.7.8.jar!/:2.7.8]
	... 86 common frames omitted

```

--- a/.github/Docker_Usage.md
+++ b/.github/Docker_Usage.md
@@ -1,36 +1,41 @@
 
 #### Using docker-compose
  
-Please fill these lines in file docker-compose.yml with correct data(mandatory):
- - spring.datasource.username=root (may be not filled if this field already filled in application.properties)
- - spring.datasource.password=     (may be not filled if this field already filled in application.properties)
+Please fill these lines in file `docker-compose.yml` with correct data (mandatory).
+In section `lpvs`:
+ - spring.datasource.username=root `(may be not filled if this field already filled in application.properties)`
+ - spring.datasource.password=     `(may be not filled if this field already filled in application.properties)`
  - github.login=
  - github.token=
+
+In section `mysqldb`:
  - MYSQL_ROOT_PASSWORD:
  
- Take in attention that value in spring.datasource.password= for user root must be the same as in the field  MYSQL_ROOT_PASSWORD.
+ Take in attention that value in `spring.datasource.password=` for user `root` must be the same as in the field  `MYSQL_ROOT_PASSWORD`.
  
-Other lines in 2 sections environment may be changed(optionally).
+Other lines in `environment` sections may be changed (optionally).
 
 Also directory of keeping MySQL data may be changed, line:
+```
  - ./mysql-lpvs-data:/var/lib/mysql # db storage -by default it is directory in root of repository with name mysql-lpvs-data
+```
 
-Start docker-compose (for old version of docker-compose):
+Start docker-compose:
+- for old version of docker-compose
    ```bash
    docker-compose up -d --build
    ```
-Start docker-compose (for new version of docker-compose):
+- for new version of docker-compose
    ```bash
    docker compose up
    ```   
    
-   
- Stop docker-compose(for old version of docker-compose):
+Stop docker-compose:
+- for old version of docker-compose:
    ```bash
    docker-compose down -v --rmi local
    ```  
-  Stop docker-compose(for new version of docker-compose):
+- for new version of docker-compose:
    ```bash
    docker compose down
-   ```  
- 
+   ``` 

--- a/.github/Docker_Usage.md
+++ b/.github/Docker_Usage.md
@@ -1,33 +1,28 @@
 
 #### Using docker-compose
  
-Please fill all needed lines in file `application-properties` according to README.md instructions.
+Please fill all needed lines in file `application-properties` according to `README.md` instructions.
 
-If you would like to use another than root user for DB that reflecting in fields:
-
+If you would like to use another than `root` user for DB that reflecting in fields in file `application.properties`:
+```
  spring.datasource.username=user
- 
  spring.datasource.password=password  
- 
- in file application.properties,
- 
- needed to add 2 fields in docker-compose.yml file in section environment before or after MYSQL_ROOT_PASSWORD values:
- 
+```
+ needed to add two fields in `docker-compose.yml` file in section `environment` before or after `MYSQL_ROOT_PASSWORD` value:
+```
   -MYSQL_USER: user
- 
   -MYSQL_PASSWORD: password
- 
-Else if only root used- i.e.
-
+```
+Otherwise, if only `root` user is used
+```
  spring.datasource.username=root
- 
  spring.datasource.password=rootpassword  
- 
-than only need to fill field
-
+```
+than only need to fill one field
+```
   -MYSQL_ROOT_PASSWORD:rootpassword
-
- But in both variants MYSQL_ROOT_PASSWORD field need to filled.
+```
+ But in both cases `MYSQL_ROOT_PASSWORD` need to be filled.
  
  
 Also directory of keeping MySQL data may be changed, line:

--- a/.github/Docker_Usage.md
+++ b/.github/Docker_Usage.md
@@ -31,6 +31,6 @@ Start docker-compose (for new version of docker-compose):
    ```  
   Stop docker-compose(for new version of docker-compose):
    ```bash
-   docker-compose down -v --rmi local
+   docker compose down
    ```  
  

--- a/.github/Docker_Usage.md
+++ b/.github/Docker_Usage.md
@@ -1,65 +1,36 @@
-# Additional information and tips for Docker usage
 
-#### There are three ways to execute the Docker container with LPVS after the image was built by command `docker build -t lpvs .` :
- 
-1. Terminal mode: the log is shown in the terminal in real-time, but the terminal must be open for running the application in the container:
-   
-   ```bash
-   docker run -p 7896:7896 --name lpvs lpvs:latest 
-   ```
- 
-2. Background mode: the container is not going to be restarted after reboot, to see the log additional commands should be used: `docker logs -f lpvs`:
-
-    ```bash
-    docker run -d -p 7896:7896 --name lpvs lpvs:latest
-    ```
-
-3. Background mode with constant usage: the container is going to be restarted after the reboot (other behavior is similar to the background mode):
-
-    ```bash
-    docker run -d -p 7896:7896 --restart unless-stopped --name lpvs lpvs:latest
-    ```
-
-  ***It is better (for disk space economy) to stop and start the same container which is created by the command `docker run`.*** 
-
-#### Useful Docker commands
-    
-To stop the running container use the following commands:
-    
-   ```bash
-   docker stop lpvs
-   ```
-
-To start the stopped container use the following commands:
-    
-   ```bash
-   docker start lpvs
-   ```
-
-To clean unused containers (if the command `docker run` was used a few times), _only stopped containers will be deleted_:
-    
-   ```bash
-   docker rm lpvs
-   ```
 #### Using docker-compose
  
-Please fill 2 lines in file docker-compose.yml with correct data(mandatory):
+Please fill these lines in file docker-compose.yml with correct data(mandatory):
+ - spring.datasource.username=root (may be not filled if this field already filled in application.properties)
+ - spring.datasource.password=     (may be not filled if this field already filled in application.properties)
  - github.login=
  - github.token=
+ - MYSQL_ROOT_PASSWORD:
+ 
+ Take in attention that value in spring.datasource.password= for user root must be the same as in the field  MYSQL_ROOT_PASSWORD.
  
 Other lines in 2 sections environment may be changed(optionally).
 
 Also directory of keeping MySQL data may be changed, line:
  - ./mysql-lpvs-data:/var/lib/mysql # db storage -by default it is directory in root of repository with name mysql-lpvs-data
 
-Start docker-compose:
+Start docker-compose (for old version of docker-compose):
    ```bash
    docker-compose up -d --build
    ```
+Start docker-compose (for new version of docker-compose):
+   ```bash
+   docker compose up
+   ```   
    
- Stop docker-compose:
+   
+ Stop docker-compose(for old version of docker-compose):
    ```bash
    docker-compose down -v --rmi local
    ```  
- 
+  Stop docker-compose(for new version of docker-compose):
+   ```bash
+   docker-compose down -v --rmi local
+   ```  
  

--- a/.github/Docker_Usage.md
+++ b/.github/Docker_Usage.md
@@ -1,20 +1,35 @@
 
 #### Using docker-compose
  
-Please fill these lines in file `docker-compose.yml` with correct data (mandatory).
-In section `lpvs`:
- - spring.datasource.username=root `(may be not filled if this field already filled in application.properties)`
- - spring.datasource.password=     `(may be not filled if this field already filled in application.properties)`
- - github.login=
- - github.token=
+Please fill all needed lines in file `application-properties` according to README.md instructions.
 
-In section `mysqldb`:
- - MYSQL_ROOT_PASSWORD:
- 
- Take in attention that value in `spring.datasource.password=` for user `root` must be the same as in the field  `MYSQL_ROOT_PASSWORD`.
- 
-Other lines in `environment` sections may be changed (optionally).
+If you would like to use another than root user for DB that reflecting in fields:
 
+ spring.datasource.username=user
+ 
+ spring.datasource.password=password  
+ 
+ in file application.properties,
+ 
+ needed to add 2 fields in docker-compose.yml file in section environment before or after MYSQL_ROOT_PASSWORD values:
+ 
+  -MYSQL_USER: user
+ 
+  -MYSQL_PASSWORD: password
+ 
+Else if only root used- i.e.
+
+ spring.datasource.username=root
+ 
+ spring.datasource.password=rootpassword  
+ 
+than only need to fill field
+
+  -MYSQL_ROOT_PASSWORD:rootpassword
+
+ But in both variants MYSQL_ROOT_PASSWORD field need to filled.
+ 
+ 
 Also directory of keeping MySQL data may be changed, line:
 ```
  - ./mysql-lpvs-data:/var/lib/mysql # db storage -by default it is directory in root of repository with name mysql-lpvs-data

--- a/README.md
+++ b/README.md
@@ -105,7 +105,8 @@ LPVS will start scanning automatically, then provide comments about the licenses
    ```
    Tip: For for usual GitHub account use  https://api.github.com  in field: github.api.url.  
    
-   For convience adding these properties(associated with github) included in docker-compose.yml (so not needed to add it in application.properties).
+   For convience adding these properties(associated with github) included in docker-compose.yml (so not needed to add it in application.properties if it added in docker-compose.yml file).
+   
    For additional information about using Docker and tips, please check file [Docker_Usage](.github/Docker_Usage.md).
    
    ```text
@@ -124,7 +125,8 @@ LPVS will start scanning automatically, then provide comments about the licenses
    spring.datasource.username=
    spring.datasource.password=
     ```
-   For convience adding these properties(DB configuration) included in docker-compose.yml (so not needed to add it in application.properties).
+   For convience adding these properties(DB configuration) included in docker-compose.yml (so not needed to add it in application.properties if it added in docker-compose.yml file ).
+   
    For additional information about using Docker and tips, please check file [Docker_Usage](.github/Docker_Usage.md).
 
    Alternatively, you can supply all the necessary values associated with GitHub and license using these env variables:
@@ -146,16 +148,18 @@ Tip: please use for this parameter -Dgithub.secret=  use LPVS as my-secret.
    Or alternatively build and run the Docker container with LPVS.
    
    For old version of Docker compose:
-       ```bash
-    docker-compose up -d --build
+    ```
+      docker-compose up -d --build
     ```
     
    For new version of Docker compose:
-   ```bash
-    docker compose up 
     ```
+     docker compose up 
+    ```
+   
+   
+For additional information about using Docker and tips, please check file [Docker_Usage](.github/Docker_Usage.md).
     
-    For additional information about using Docker and tips, please check file [Docker_Usage](.github/Docker_Usage.md).
     
 5. Install [ngrok](https://dashboard.ngrok.com/get-started) (step 1 and 2) and run it with the following command:
     ```bash

--- a/README.md
+++ b/README.md
@@ -50,8 +50,52 @@ LPVS will start scanning automatically, then provide comments about the licenses
     pip3 install scanoss
     ```
 
-2. Create necessary MySQL database and user and fill in `licenses` and `license_conflicts` tables with the information about permitted, restricted, and prohibited licenses (mandatory) as well as their compatibility specifics (optional). 
-An example database dump file can be found in the repository at `src/main/resources/database_dump.sql`.
+2. Create necessary MySQL database and user (for the case when the database is supposed to be used)
+
+    2.1 Install MySQL server locally:
+
+    ```
+    sudo apt install mysql-server
+    ```
+
+    2.2 Start server:
+
+    ```
+    sudo service mysql start
+    ```
+
+    2.3 Then open Mysql command line interface:
+
+    ```
+    sudo mysql
+    ```
+
+    Run the following commands to create necessary database and user for SOSHUB:
+
+    ```
+    mysql> create database lpvs;
+    mysql> create user username;
+    mysql> grant all on lpvs.* to username;
+    mysql> alter user username identified by identified by 'password';
+    mysql> exit;
+    ```
+
+    2.4 Import existing dump file to newly created databse (optional instruction below):
+    ```
+    mysql -u username -p lpvs < src/main/resources/database_dump.sql
+    ```
+
+    When prompted, enter password: `password`
+
+    2.5 Fill in `licenses` and `license_conflicts` tables with the information about permitted, restricted, and prohibited licenses (mandatory) as well as their compatibility specifics (optional). 
+
+    An example database dump file can be found in the repository at `src/main/resources/database_dump.sql`.
+
+    2.6 Update the lines of the `src/main/resources/application.properties` file:
+    ```text
+    spring.datasource.username=username
+    spring.datasource.password=password
+    ```
 
 3. Fill in the lines of the `src/main/resources/application.properties` file:
     ```text
@@ -63,8 +107,6 @@ An example database dump file can be found in the repository at `src/main/resour
    ```
    For convience adding these properties(associated with github) included in docker-compose.yml (so not needed to add it in application.properties).
    For additional information about using Docker and tips, please check file [Docker_Usage](.github/Docker_Usage.md).
-   
-   
    
    ```text
    # Used license scanner: scanoss (at the moment, only this scanner is supported)
@@ -84,8 +126,6 @@ An example database dump file can be found in the repository at `src/main/resour
     ```
    For convience adding these properties(DB configuration) included in docker-compose.yml (so not needed to add it in application.properties).
    For additional information about using Docker and tips, please check file [Docker_Usage](.github/Docker_Usage.md).
-
-
 
    Alternatively, you can supply all the necessary values associated with GitHub and license using these env variables:
    `LPVS_GITHUB_LOGIN`, `LPVS_GITHUB_TOKEN`, `LPVS_GITHUB_API_URL`, `LPVS_GITHUB_SECRET` and `LPVS_LICENSE_CONFLICT`.

--- a/README.md
+++ b/README.md
@@ -103,9 +103,9 @@ LPVS will start scanning automatically, then provide comments about the licenses
    github.api.url=
    github.secret=LPVS
    ```
-   Tip: For for usual GitHub account use  https://api.github.com  in field: github.api.url.  
+   Tip: For personal GitHub account use  `https://api.github.com`  in field `github.api.url=`.  
    
-   For convience adding these properties(associated with github) included in docker-compose.yml (so not needed to add it in application.properties if it added in docker-compose.yml file).
+   For convenience, adding these properties (associated with GitHub) included in `docker-compose.yml` (so not needed to add them in `application.properties` if added in `docker-compose.yml` file).
    
    For additional information about using Docker and tips, please check file [Docker_Usage](.github/Docker_Usage.md).
    
@@ -143,19 +143,19 @@ LPVS will start scanning automatically, then provide comments about the licenses
    ```bash
    java -jar -Dgithub.token=<`my-token`> -Dgithub.secret=<`my-secret`> lpvs-1.0.2.jar
    ```
-Tip: please use for this parameter -Dgithub.secret=  use LPVS as my-secret.
+   Tip: for parameter `-Dgithub.secret=`  is recommended to use `LPVS` as `my-secret`.
 
    Or alternatively build and run the Docker container with LPVS.
    
    For old version of Docker compose:
     ```
-      docker-compose up -d --build
+    docker-compose up -d --build
     ```
     
    For new version of Docker compose:
-    ```
-     docker compose up 
-    ```
+   ```
+   docker compose up 
+   ```
    
    
 For additional information about using Docker and tips, please check file [Docker_Usage](.github/Docker_Usage.md).

--- a/README.md
+++ b/README.md
@@ -82,11 +82,9 @@ LPVS will start scanning automatically, then provide comments about the licenses
 
     2.4 Import existing dump file to newly created databse (optional instruction below):
     ```
-    mysql -u username -p lpvs < src/main/resources/database_dump.sql
+    mysql -u[username] -p[password] < src/main/resources/database_dump.sql
     ```
-
-    When prompted, enter password: `password`
-
+    
     2.5 Fill in `licenses` and `license_conflicts` tables with the information about permitted, restricted, and prohibited licenses (mandatory) as well as their compatibility specifics (optional). 
 
     An example database dump file can be found in the repository at `src/main/resources/database_dump.sql`.
@@ -105,6 +103,8 @@ LPVS will start scanning automatically, then provide comments about the licenses
    github.api.url=
    github.secret=LPVS
    ```
+   Tip: For for usual GitHub account use  https://api.github.com  in field: github.api.url.  
+   
    For convience adding these properties(associated with github) included in docker-compose.yml (so not needed to add it in application.properties).
    For additional information about using Docker and tips, please check file [Docker_Usage](.github/Docker_Usage.md).
    
@@ -141,12 +141,20 @@ LPVS will start scanning automatically, then provide comments about the licenses
    ```bash
    java -jar -Dgithub.token=<`my-token`> -Dgithub.secret=<`my-secret`> lpvs-1.0.2.jar
    ```
+Tip: please use for this parameter -Dgithub.secret=  use LPVS as my-secret.
 
-   Or alternatively build and run the Docker container with LPVS:
-   ```bash
-    docker build -t lpvs .
-    docker run -p 7896:7896 --name lpvs -e LPVS_GITHUB_TOKEN=<`github.token`> -e LPVS_GITHUB_SECRET=<`github.secret`> lpvs:latest
+   Or alternatively build and run the Docker container with LPVS.
+   
+   For old version of Docker compose:
+       ```bash
+    docker-compose up -d --build
     ```
+    
+   For new version of Docker compose:
+   ```bash
+    docker compose up 
+    ```
+    
     For additional information about using Docker and tips, please check file [Docker_Usage](.github/Docker_Usage.md).
     
 5. Install [ngrok](https://dashboard.ngrok.com/get-started) (step 1 and 2) and run it with the following command:

--- a/README.md
+++ b/README.md
@@ -105,8 +105,6 @@ LPVS will start scanning automatically, then provide comments about the licenses
    ```
    Tip: For personal GitHub account use  `https://api.github.com`  in field `github.api.url=`.  
    
-   For convenience, adding these properties (associated with GitHub) included in `docker-compose.yml` (so not needed to add them in `application.properties` if added in `docker-compose.yml` file).
-   
    For additional information about using Docker and tips, please check file [Docker_Usage](.github/Docker_Usage.md).
    
    ```text
@@ -125,7 +123,6 @@ LPVS will start scanning automatically, then provide comments about the licenses
    spring.datasource.username=
    spring.datasource.password=
     ```
-   For convience adding these properties(DB configuration) included in docker-compose.yml (so not needed to add it in application.properties if it added in docker-compose.yml file ).
    
    For additional information about using Docker and tips, please check file [Docker_Usage](.github/Docker_Usage.md).
 

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -8,6 +8,8 @@ services:
     restart: always
     ports:
       - "7896:7896"
+    environment:
+      - spring.datasource.url=jdbc:mysql://mysqldb:3306/lpvs
     depends_on:
       mysqldb:
         condition: service_healthy

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -8,16 +8,6 @@ services:
     restart: always
     ports:
       - "7896:7896"
-    environment:
-      ## spring data
-      - spring.datasource.url=jdbc:mysql://mysqldb:3306/lpvs
-      - spring.datasource.username=root
-      - spring.datasource.password=lpvsroot
-      ## github data
-      - github.login=
-      - github.token=
-      - github.api.url=https://api.github.com  # for usual GitHub account 
-      - github.secret=LPVS 
     depends_on:
       mysqldb:
         condition: service_healthy

--- a/src/main/java/com/lpvs/repository/LPVSLicenseConflictRepository.java
+++ b/src/main/java/com/lpvs/repository/LPVSLicenseConflictRepository.java
@@ -18,10 +18,10 @@ import java.util.List;
 @Repository
 public interface LPVSLicenseConflictRepository extends CrudRepository<LPVSLicenseConflict, Long> {
 
-    @Query(value = "SELECT * FROM lpvs_license_conflicts", nativeQuery = true)
+    @Query(value = "SELECT * FROM license_conflicts", nativeQuery = true)
     List<LPVSLicenseConflict> takeAllLicenseConflicts();
 
-    @Query(value = "SELECT * FROM lpvs_license_conflicts WHERE (lpvs_license_conflicts.repository_license_id = :license1 AND lpvs_license_conflicts.conflict_license_id = :license2) " +
-            "OR (lpvs_license_conflicts.repository_license_id = :license2 AND lpvs_license_conflicts.conflict_license_id = :license1) ORDER BY id DESC LIMIT 1", nativeQuery = true)
+    @Query(value = "SELECT * FROM license_conflicts WHERE (license_conflicts.repository_license_id = :license1 AND license_conflicts.conflict_license_id = :license2) " +
+            "OR (license_conflicts.repository_license_id = :license2 AND license_conflicts.conflict_license_id = :license1) ORDER BY id DESC LIMIT 1", nativeQuery = true)
     LPVSLicenseConflict findLicenseConflict(@Param("license1") Long license1, @Param("license2") Long license2);
 }

--- a/src/main/java/com/lpvs/repository/LPVSLicenseRepository.java
+++ b/src/main/java/com/lpvs/repository/LPVSLicenseRepository.java
@@ -18,12 +18,12 @@ import java.util.List;
 @Repository
 public interface LPVSLicenseRepository extends CrudRepository<LPVSLicense, Long> {
 
-    @Query(value = "SELECT * FROM code_licenses", nativeQuery = true)
+    @Query(value = "SELECT * FROM licenses", nativeQuery = true)
     List<LPVSLicense> takeAllLicenses();
 
-    @Query(value =  "SELECT * FROM code_licenses WHERE code_licenses.license_spdx = :spdxId ORDER BY id DESC LIMIT 1", nativeQuery = true)
+    @Query(value =  "SELECT * FROM licenses WHERE licenses.license_spdx = :spdxId ORDER BY id DESC LIMIT 1", nativeQuery = true)
     LPVSLicense searchBySpdxId(@Param("spdxId") String spdxId);
 
-    @Query(value = "SELECT * FROM code_licenses WHERE code_licenses.license_alternative_names LIKE %:licenseName% ORDER BY id DESC LIMIT 1", nativeQuery = true)
+    @Query(value = "SELECT * FROM licenses WHERE licenses.license_alternative_names LIKE %:licenseName% ORDER BY id DESC LIMIT 1", nativeQuery = true)
     LPVSLicense searchByAlternativeLicenseNames(@Param("licenseName") String licenseName);
 }

--- a/src/main/java/com/lpvs/repository/LPVSQueueRepository.java
+++ b/src/main/java/com/lpvs/repository/LPVSQueueRepository.java
@@ -15,6 +15,6 @@ import java.util.List;
 
 public interface LPVSQueueRepository extends JpaRepository<LPVSQueue, Long> {
 
-    @Query(value = "SELECT * FROM lpvs_queue", nativeQuery = true)
+    @Query(value = "SELECT * FROM queue", nativeQuery = true)
     List<LPVSQueue> getQueueList();
 }

--- a/src/main/java/com/lpvs/service/LPVSQueueService.java
+++ b/src/main/java/com/lpvs/service/LPVSQueueService.java
@@ -41,9 +41,6 @@ public class LPVSQueueService {
 
     private int maxAttempts;
 
-    @Value("${soshub.url}")
-    private String SOSHUB_URL;
-
     @Autowired
     public LPVSQueueService(LPVSGitHubService gitHubService,
                             LPVSDetectService detectService,
@@ -165,12 +162,10 @@ public class LPVSQueueService {
                     List<LPVSLicenseService.Conflict<String, String>> detectedConflicts = licenseService.findConflicts(webhookConfig, files);
 
                     log.debug("Creating comment");
-                    webhookConfig.setHubLink(SOSHUB_URL + "/view/lpvs/" + pullRequest.getId());
                     gitHubService.commentResults(webhookConfig, files, detectedConflicts, pullRequest);
                     log.debug("Results posted on GitHub");
                 } else {
                     log.warn("Files are not found. Probably pull request is not exists.");
-                    webhookConfig.setHubLink(SOSHUB_URL + "/view/lpvs/" + pullRequest.getId());
                     gitHubService.commentResults(webhookConfig, null, null, pullRequest);
                     delete(webhookConfig);
                     throw new Exception("Files are not found. Probably pull request is not exists. Terminating.");
@@ -181,7 +176,6 @@ public class LPVSQueueService {
             pullRequest = lpvsPullRequestRepository.saveAndFlush(pullRequest);
             log.error("Can't authorize commentResults() " + e);
             e.printStackTrace();
-            webhookConfig.setHubLink(SOSHUB_URL + "/view/lpvs/" + pullRequest.getId());
             delete(webhookConfig);
         }
     }

--- a/src/main/resources/application.properties
+++ b/src/main/resources/application.properties
@@ -18,7 +18,7 @@ github.login=
 # Corresponding env. variable LPVS_GITHUB_TOKEN
 github.token=
 # Corresponding env. variable LPVS_GITHUB_API_URL
-github.api.url=https://api.github.com                #use for usual GitHub account 
+github.api.url=https://api.github.com
 # Corresponding env. variable LPVS_GITHUB_SECRET
 github.secret=LPVS
 

--- a/src/main/resources/application.properties
+++ b/src/main/resources/application.properties
@@ -14,13 +14,13 @@ license_conflict=db
 
 # GitHub settings
 # Corresponding env. variable LPVS_GITHUB_LOGIN
-#github.login=
+github.login=
 # Corresponding env. variable LPVS_GITHUB_TOKEN
-#github.token=
+github.token=
 # Corresponding env. variable LPVS_GITHUB_API_URL
-#github.api.url=
+github.api.url=https://api.github.com                #use for usual GitHub account 
 # Corresponding env. variable LPVS_GITHUB_SECRET
-#github.secret=LPVS
+github.secret=LPVS
 
 # Used Core Pool Size
 lpvs.cores=8

--- a/src/main/resources/database_dump.sql
+++ b/src/main/resources/database_dump.sql
@@ -17,6 +17,35 @@ USE `lpvs`;
 /*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
 /*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
 
+
+--
+-- Table structure for table `licenses`
+--
+
+DROP TABLE IF EXISTS `licenses`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `licenses` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `license_usage` varchar(255) DEFAULT NULL,
+  `license_name` varchar(255) NOT NULL,
+  `license_spdx` varchar(255) NOT NULL,
+  `license_alternative_names` longtext DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `spdx_id` (`license_spdx`)
+) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `licenses`
+--
+
+LOCK TABLES `licenses` WRITE;
+/*!40000 ALTER TABLE `licenses` DISABLE KEYS */;
+INSERT INTO `licenses` VALUES (1,'PERMITTED','Apache License 2.0','Apache-2.0',''),(2,'PROHIBITED','GNU General Public License v3.0 only','GPL-3.0-only',''),(3,'PERMITTED','OpenSSL License','OpenSSL',''),(4,'RESTRICTED','GNU Lesser General Public License v2.1 or later','GPL-2.0-or-later',''),(5,'PERMITTED','MIT License','MIT','');
+/*!40000 ALTER TABLE `licenses` ENABLE KEYS */;
+UNLOCK TABLES;
+
 --
 -- Table structure for table `license_conflicts`
 --
@@ -26,13 +55,13 @@ DROP TABLE IF EXISTS `license_conflicts`;
 /*!50503 SET character_set_client = utf8mb4 */;
 CREATE TABLE `license_conflicts` (
   `id` bigint(20) NOT NULL AUTO_INCREMENT,
-  `conflict_license` bigint(20) NOT NULL,
-  `repository_license` bigint(20) NOT NULL,
+  `conflict_license_id` bigint(20) NOT NULL,
+  `repository_license_id` bigint(20) NOT NULL,
   PRIMARY KEY (`id`),
-  KEY `FKrtmiqvoah4x5lprxu186vp0i` (`conflict_license`),
-  KEY `FKbgosahkomx4ns2e7jyics99cl` (`repository_license`),
-  CONSTRAINT `FKbgosahkomx4ns2e7jyics99cl` FOREIGN KEY (`repository_license`) REFERENCES `licenses` (`id`),
-  CONSTRAINT `FKrtmiqvoah4x5lprxu186vp0i` FOREIGN KEY (`conflict_license`) REFERENCES `licenses` (`id`)
+  KEY `conflictlicense_idx` (`conflict_license_id`),
+  KEY `repositorylicense_idx` (`repository_license_id`),
+  CONSTRAINT `conflictlicense` FOREIGN KEY (`conflict_license_id`) REFERENCES `licenses` (`id`),
+  CONSTRAINT `repositorylicense` FOREIGN KEY (`repository_license_id`) REFERENCES `licenses` (`id`)
 ) ENGINE=InnoDB AUTO_INCREMENT=4 DEFAULT CHARSET=latin1;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
@@ -47,31 +76,86 @@ INSERT INTO `license_conflicts` VALUES (1,4,1),(2,4,3),(3,4,5);
 UNLOCK TABLES;
 
 --
--- Table structure for table `licenses`
+-- Table structure for table `pull_requests`
 --
 
-DROP TABLE IF EXISTS `licenses`;
+DROP TABLE IF EXISTS `pull_requests`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!50503 SET character_set_client = utf8mb4 */;
-CREATE TABLE `licenses` (
+CREATE TABLE `pull_requests` (
   `id` bigint(20) NOT NULL AUTO_INCREMENT,
-  `license_usage` varchar(255) DEFAULT NULL,
-  `license_name` varchar(255) NOT NULL,
-  `license_spdx` varchar(255) NOT NULL,
-  PRIMARY KEY (`id`),
-  UNIQUE KEY `spdx_id` (`license_spdx`)
-) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=latin1;
+  `scan_date` datetime(6) NOT NULL,
+  `user` varchar(255) DEFAULT NULL,
+  `repository_name` varchar(255) NOT NULL,
+  `url` longtext NOT NULL,
+  `diff_url` longtext,
+  `status` varchar(255) DEFAULT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=latin1;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
 --
--- Dumping data for table `licenses`
+-- Table structure for table `detected_license`
 --
 
-LOCK TABLES `licenses` WRITE;
-/*!40000 ALTER TABLE `licenses` DISABLE KEYS */;
-INSERT INTO `licenses` VALUES (1,'PERMITTED','Apache License 2.0','Apache-2.0'),(2,'PROHIBITED','GNU General Public License v3.0 only','GPL-3.0-only'),(3,'PERMITTED','OpenSSL License','OpenSSL'),(4,'RESTRICTED','GNU Lesser General Public License v2.1 or later','GPL-2.0-or-later'),(5,'PERMITTED','MIT License','MIT');
-/*!40000 ALTER TABLE `licenses` ENABLE KEYS */;
-UNLOCK TABLES;
+DROP TABLE IF EXISTS `detected_license`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `detected_license` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `pull_request_id` bigint(20) DEFAULT NULL,
+  `license_id` bigint(20) DEFAULT NULL,
+  `conflict_id` bigint(20) DEFAULT NULL,
+  `repository_license_id` bigint(20) DEFAULT NULL,
+  `file_path` longtext,
+  `match_type` varchar(255) DEFAULT NULL,
+  `match_value` varchar(255) DEFAULT NULL,
+  `match_lines` varchar(255) DEFAULT NULL,
+  `component_file_path` longtext,
+  `component_file_url` longtext,
+  `component_name` varchar(255) DEFAULT NULL,
+  `component_lines` varchar(255) DEFAULT NULL,
+  `component_url` longtext,
+  `component_version` varchar(255) DEFAULT NULL,
+  `component_vendor` varchar(255) DEFAULT NULL,
+  `issue` bit(1) DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  KEY `pullrequestid_idx` (`pull_request_id`),
+  KEY `licenseid_idx` (`license_id`),
+  KEY `repolicenseid_idx` (`repository_license_id`),
+  KEY `conflictid_idx` (`conflict_id`),
+  CONSTRAINT `conflictid` FOREIGN KEY (`conflict_id`) REFERENCES `license_conflicts` (`id`),
+  CONSTRAINT `licenseid` FOREIGN KEY (`license_id`) REFERENCES `licenses` (`id`),
+  CONSTRAINT `pullrequestid` FOREIGN KEY (`pull_request_id`) REFERENCES `pull_requests` (`id`),
+  CONSTRAINT `repolicenseid` FOREIGN KEY (`repository_license_id`) REFERENCES `licenses` (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `queue`
+--
+
+DROP TABLE IF EXISTS `queue`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `queue` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `action` bigint(20) NOT NULL,
+  `attempts` int(11) DEFAULT '0',
+  `scan_date` datetime(6) DEFAULT NULL,
+  `user` varchar(255) DEFAULT NULL,
+  `review_system_type` varchar(255) DEFAULT NULL,
+  `repository_url` longtext,
+  `pull_request_url` longtext,
+  `pull_request_api_url` longtext,
+  `pull_request_diff_url` longtext,
+  `status_callback_url` longtext,
+  `commit_sha` varchar(255) DEFAULT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+
 /*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;
 
 /*!40101 SET SQL_MODE=@OLD_SQL_MODE */;

